apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.ejs

```diff
@@ -0,0 +1,276 @@
+<div class="modal js-rubric-settings-modal" tabindex="-1" role="dialog">
+  <div class="modal-dialog modal-xl border-info" role="document">
+    <form name="rubric-settings" method="POST" class="needs-validation"
+          data-max-points="<%= assessment_question.max_points %>">
+      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+      <input type="hidden" name="__action" value="modify_rubric_settings">
+      <input type="hidden" name="modified_at" value="<%= locals.rubric_data?.modified_at %>">
+      <input type="hidden" name="use_rubric" value="true">
+      
+      <div class="modal-content">
+        <div class="modal-header bg-info text-light">
+          <h5 class="modal-title">Rubric settings</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <% if (assessment_question.max_auto_points) { %>
+          <div class="form-row">
+            <div class="col-6">
+              <div class="form-check">
+                <label class="form-check-label">
+                  <input class="form-check-input js-replace-auto-points-input"
+                         name="replace_auto_points"
+                         type="radio" value="true" required
+                         data-max-points="<%= assessment_question.max_points %>"
+                         <% if (locals.rubric_data?.replace_auto_points ?? !assessment_question.max_manual_points) { %>checked<% } %>
+                         aria-describedby="rubric-settings-replace-auto-explanation">
+                  Apply rubric to total points (ignore auto points)
+                </label>
+              </div>
+            </div>
+            <div class="col-6">
+              <div class="form-check">
+                <label class="form-check-label">
+                  <input class="form-check-input js-replace-auto-points-input" name="replace_auto_points"
+                         type="radio" value="false" required
+                         data-max-points="<%= assessment_question.max_manual_points %>"
+                         <% if (!(locals.rubric_data?.replace_auto_points ?? !assessment_question.max_manual_points)) { %>checked<% } %>
+                         aria-describedby="rubric-settings-replace-auto-explanation">
+                  Apply rubric to manual points (keep auto points)
```

Comment:
Because this is the default option (and the "normal" method of operation) I think it should be first?

Reply:
Done in 61330bb5f. Also added the max points for each option for clarity.